python/console/console_editor.py

```diff
@@ -55,6 +55,56 @@
 from qgis.utils import OverrideCursor
 
 
+def findMinimalDistanceIndex(source, target):
+    """ Find the source substring index that most closely matches the target string"""
+    index = min(len(source), len(target))
+
+    # Fallback to difflib if Levenshtein is not available
```

Comment:
we have a native levenshtein distance function in QgsStringUtils::levenshteinDistance. Can this be used instead of the python library?

Reply:
Yes, absolutely. I hadn't encountered `QgsStringUtils` yet.